File: app/src/main/java/com/bitcoin/merchant/app/network/NetworkStateReceiver.java
package com.bitcoin.merchant.app.network;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import androidx.localbroadcastmanager.content.LocalBroadcastManager;

import com.bitcoin.merchant.app.Action;
import com.bitcoin.merchant.app.MainActivity;

/**
 * Created by riaanvos on 11/12/15.
 */
public class NetworkStateReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(final Context context, final Intent intent) {
        if (intent.getExtras() != null) {
            //TODO reconnect
           // LocalBroadcastManager.getInstance(context).sendBroadcastSync(new Intent(Action.ACTION_INTENT_RECONNECT));
        }
    }
}

File: app/src/main/java/com/bitcoin/merchant/app/network/NetworkStateReceiver.kt
package com.bitcoin.merchant.app.network

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

/**
 * Created by riaanvos on 11/12/15.
 */
class NetworkStateReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        if (intent.extras != null) { //TODO reconnect
// LocalBroadcastManager.getInstance(context).sendBroadcastSync(new Intent(Action.ACTION_INTENT_RECONNECT));
        }
    }
}